server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -204,6 +204,8 @@
 
     private static final String DATA_BLOB_PREFIX = "__";
 
+    private final Settings globalSettings;
```

Comment:
can we just call this `settings`?

Reply:
The rest of the class calls it `globalSettings`. I'll just rename it everywhere. So the class lines up with the rest of our classes.